dev-docs/howtoguides/how_to_release_a_new_version_of_ua.md

```diff
@@ -126,26 +129,26 @@ If this is your first time releasing ubuntu-advantage-tools, you'll need to do t
     d. `git rebase --onto pkg/ubuntu/devel <last-version-tag> <this-version-tag>`
       * e.g. `git rebase --onto pkg/ubuntu/devel 27.0.2 27.1`
       * You may need to resolve conflicts, but hopefully these will be minimal.
-      * You'll end up in a detached state
+      * You'll end up in a detached state.
 
     e. `git checkout -B upload-<this-version>-kinetic`
-      * This creates a new local branch name based on your detached branch
+      * This creates a new local branch name based on your detached branch.
 
-    f. Make sure the changelog version contains the release version in the name (For example, `27.1~22.10.1`)
+    f. Make sure the changelog version contains the release version in the name (e.g., `27.1~22.10.1`)
 
     g. `git push <your_launchpad_user> upload-<this-version>-kinetic`
 
     h. On Launchpad, create a merge proposal for this version which targets `ubuntu/devel`
-      * For an example, see the [27.9 merge proposal](https://code.launchpad.net/~orndorffgrant/ubuntu/+source/ubuntu-advantage-tools/+git/ubuntu-advantage-tools/+merge/422906)
-      * Add 2 review slots for `canonical-server-reporter` and `canonical-server-core-reviewers`
+      * For an example, see the [27.9 merge proposal](https://code.launchpad.net/~orndorffgrant/ubuntu/+source/ubuntu-advantage-tools/+git/ubuntu-advantage-tools/+merge/422906).
+      * Add 2 review slots for `canonical-server-reporter` and `canonical-server-core-reviewers`.
 
 4. Server Team Review and Pre-SRU Review
 
-    a. Ask the assigned ubuntu-advantage-tools reviewer/sponsor from Server team for a review of your MPs (If you don't know who that is, ask in ~Server). Include a link to the MP into ubuntu/devel and to the SRU bug.
+    a. Ask the assigned ubuntu-advantage-tools reviewer/sponsor from Server team for a review of your MPs. If you don't know who that is, ask in ~Server. Include a link to the MP and to the SRU bug in ubuntu/devel.
```

Comment:
This edit changes the meaning slightly, but the old version is confusing. Here's an alternative suggestion
```suggestion
    a. Ask the assigned ubuntu-advantage-tools reviewer/sponsor from Server team for a review of your MPs. If you don't know who that is, ask in ~Server. Include a link to the ubuntu/devel MP and to the SRU bug.
```